components/automate-deployment/pkg/server/reindex-start.go

```diff
@@ -0,0 +1,37 @@
+package server
```

Comment:
Any specific reason for naming this file as reindex-start.go? If not please rename it to reindex.go